app/views/recipes/general_shopping_list.html.erb

```diff
@@ -0,0 +1,23 @@
+<section class="gs-section">
+  <div class='gs-container'>
+    <h2 class="gs-title">Shopping List</h2>
+    <div class="gs-info">
+      <h3>Amount of food items to buy: <%= @foods.count %></h3>
+      <h3>Total value of food needed: <%= @foods.sum { |r| r.recipe_foods.sum {|f| f[:quantity]*r.price }} %></h3>

```

Comment:
- [ ] I would recommend using a helper to apply logic and from `view` you can call this function to keep `view` clean.

Reply:
We follow the recomendations and now this is how that line looks like

https://github.com/miguelesco/recipe_app/blob/ff0a02b135bff807738a73a2f6efc3f7affb6e97/app/views/general_shopping_list/index.html.erb#L2

Reply:
- [ ] Still, the logic is there, I think you have changed that line in some other file. Please check the file ` app/views/recipes/general_shopping_list.html.erb` and make sure you are doing it in the correct branch(develop).